source/tools/testConsistency/dotnetWPF-renderer/WPFRenderer/WPFRenderer/Program.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License.
+using AdaptiveCards;
+using AdaptiveCards.Rendering.Wpf;
+using AdaptiveCards.Templating;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace WPFRenderer
+{
+    public class Program
```

Comment:
again class comments here would be nice and so would a project level readme; Also we already have a WPF renderer in our SDK - can we name this to be slightly less ambiguous? Is the WPFDriver?